plugins/source/aws/client/client.go

```diff
@@ -321,7 +321,7 @@ func configureAwsClient(ctx context.Context, logger zerolog.Logger, awsConfig *S
 	// Test out retrieving credentials
 	if _, err := awsCfg.Credentials.Retrieve(ctx); err != nil {
 		logger.Error().Err(err).Msg("error retrieving credentials")
-		return awsCfg, fmt.Errorf("error retrieving credentials: %w", err)
+		return awsCfg, fmt.Errorf("error retrieving AWS credentials. Please verify your credentials and try again")
```

Comment:
```suggestion
		return awsCfg, fmt.Errorf("error retrieving AWS credentials (see logs for details). Please verify your credentials and try again")
```

I'm thinking we should just point users to the log at least, since it's likely to be new users running into this error, and they may not be aware of the log file yet.

Reply:
Yeah, the error is not great in the local profile case, but I guess it might be more enlightening in some other cases (depending on the underlying reason for failure)?